jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -12,32 +12,22 @@ pipeline {
         }
     }
     parameters {
-        string(
-            name: 'BUILD_ID',
-            description: 'Build ID of the distribution build manifest generated.',
-            trim: true
-        )
         string(
             name: 'VERSION',
-            description: 'Version of the distribution build manifest.',
+            description: 'Version of the release to download the artifacts.',
             trim: true
         )
     }
     environment {
-        BUILD_MANIFEST = "builds/opensearch/manifest.yml"
+        BUILD_MANIFEST = "opensearch-$VERSION/manifest.yml"
     }
     stages {
         stage('Create Release Tag') {
             steps {
                 script {
-                    manifest_path = "distribution-build-opensearch/${VERSION}/${BUILD_ID}/linux/x64/builds/opensearch/manifest.yml"
-                    downloadFromS3(
-                        destPath: "$WORKSPACE/$BUILD_MANIFEST",
-                        bucket: "${ARTIFACT_BUCKET_NAME}",
-                        path: "${manifest_path}",
-                        force: true
-                    )
-                    echo "Distribution build manifest is downloaded to $BUILD_MANIFEST"
+                    release_url = "https://artifacts.opensearch.org/releases/bundle/opensearch/$VERSION/opensearch-$VERSION-linux-x64.tar.gz"
+                    sh "curl -SLO $release_url && tar -xf opensearch-$VERSION-linux-x64.tar.gz"
```

Comment:
You could extract just the manifest.yml, faster.

Reply:
@zelinh want to make the above change before merging?

Reply:
Sure I will do that.

Reply:
This has been updated. I will merge this PR now then.